tsrc/cli/main.py

```diff
@@ -115,6 +115,8 @@ def main(args=None):
     message_group.add_argument("-m", "--message", dest="mr_title")
     message_group.add_argument("--wip", action="store_true", help="Mark merge request as WIP")
     message_group.add_argument("--ready", action="store_true", help="Mark merge request as ready")
+    message_group.add_argument("--approve", action="store_true", help="Approve merge request")
```

Comment:
You are adding options to the 'message' group, which only exists to make sure you can' t use
`--wip`  and `--ready`  at the same time

Instead you should create a new mutually exclusive group for the `--approve`  and `--ready` options.

Reply:
I use Gitlab EE (that's why I am interested in such a feature).

Now I come to realize that it may not be the right way, cause the one, who creates one MR, can not approve it (It must be someone else). Somehow, I believe it is linked to the conversation you have in #33.

For now, I'll fix these obvious errors and lint errors. But there may be a bigger picture worth to think about